### PR TITLE
Create roman_to_integer.md

### LINKED ISSUE #7640: Adding Roman to Integer code to Brute Force
### Description

The Roman to Integer code is a really good example, which I want to add by creating a different md file.

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Preet1406**
<hr>

## Description 📜

The Roman to Integer code is a really good example, which I wanted to add by creating a different md file.
 
Fixes #7640 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+## Roman To Integer Using Brute Force Approach
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:1]: Wrong level of heading, use #.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### Brute Force
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:3]: Wrong level of heading, use ##.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+A Brute Force Algorithm is a straight-forward approach to a problem i.e the first method or algorithm which come in our mind to solve a particular problem. In this we try every possibility rather than going for advance techniques to improve efficiency.
+
+### Roman To Integer
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:6]: Wrong level of heading, use ##.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+Task: Given a roman numeral, convert it to an integer.
+
+Approach: Roman numerals are represented by seven symbols: I, V, X, L, C, D and M.
+
+![image](https://user-images.githubusercontent.com/73405757/140963448-360c6a7f-f391-4396-8fbd-68ab3c365c52.png)
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:12]: Add reference source for the image.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add a relative link in the readme file.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 1 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md]: changed

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 2 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md]: changed

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 3 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md]: done

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 4 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:12]: added reference at last

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+Output: 1994
+
+## Time And Space Complexity
+
+Time Complexity: O(n), where n is the length of the string. 
+
+Space Complexity: O(1), as no extra space is required.
+
+## Reference:
+https://leetcode.com/problems/roman-to-integer/
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:118]: Embed the link in the text.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add reference source to image just below it.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 5 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md]: Is it fine now?

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+## Roman To Integer
+
+Task: Given a roman numeral, convert it to an integer.
+
+Approach: Roman numerals are represented by seven symbols: I, V, X, L, C, D and M.
+
+![image](https://user-images.githubusercontent.com/73405757/140963448-360c6a7f-f391-4396-8fbd-68ab3c365c52.png)
+
+[Reference source to image](https://leetcode.com/problems/roman-to-integer/)
```
Comment (kashishahuja2002) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:14]: Change this to:
"Source: [LeetCode](https://leetcode.com/problems/roman-to-integer/)"

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md (hunk 6 above)
Comment (Preet1406) [reply C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md]: Changed, please check

**Review by kashishahuja2002 — APPROVED** [review]
LGTM. Tag the supervisor of your batch for further reviewing and merging the PR.

**Comment by Preet1406** [comment]
2024-SANDHYA

**Comment by Preet1406** [comment]
@2024-SANDHYA

**Comment by Preet1406** [comment]
@Suvraneel please merge it, it has been reviewed.

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+Source: [LeetCode](https://leetcode.com/problems/roman-to-integer/)
+
+## Algorithm:  
+
+Step 1) Split the string into characters.
+
+Step 2) Convert the characters into its integer value.
+
+Step 3) Take characters starting from index 0:
```
Comment (Suvraneel) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:22]:
```suggestion
Step 1) Split the string into characters.  
Step 2) Convert the characters into its integer value.  
Step 3) Take characters starting from index 0: 
```
To add a new line, just add 2 spaces at end of previous lines

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+Step 1) Split the string into characters.
+
+Step 2) Convert the characters into its integer value.
+
+Step 3) Take characters starting from index 0: 
+* If current value of character is greater than or equal to the value of next character, add its value to the total.
+* else subtract its value by adding the value of the next character to the total.
+        
+## Code of converting Roman numerals to integer in C++
+
```
Comment (Suvraneel) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:27]:
```suggestion
        
## Code of converting Roman numerals to integer in C++  
```
Remove extra newline chars

File: C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+#### Example 2:
+Input: str = "MCMXCIV"
+
+Output: 1994
+
+## Time And Space Complexity
+
+Time Complexity: O(n), where n is the length of the string. 
+
+Space Complexity: O(1), as no extra space is required.
```
Comment (Suvraneel) [inline C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md:116]:
```suggestion
Input: str = "LVIII"  
Output: 58

#### Example 2:
Input: str = "MCMXCIV"  
Output: 1994

## Time And Space Complexity

Time Complexity: O(n), where n is the length of the string.   
Space Complexity: O(1), as no extra space is required.
```

Ditto...
Line count matters... remove extra lines

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@Preet1406

**Comment by Preet1406** [comment]
@Suvraneel changes done, please check again

**Review by Suvraneel — CHANGES_REQUESTED** [review]
This folder is better suited...
https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Number%20System/Conversion-Programs

Shift & do the needful

**Comment by Preet1406** [comment]
@Suvraneel changed the location, please check

**Comment by Suvraneel** [comment]
@Preet1406  LGTM 🚀

**Pull request merged**